addons/web/static/src/scss/fields.scss

```diff
@@ -446,6 +458,53 @@
 
 }
 
+.o_field_invalid {
+    .o_signature {
+        outline: 3px solid #DC6965;
```

Comment:
try to not hardcode colors in the styles, make them depend on an existing variable